src/pages/components/Widget.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from '@storybook/react'
-import { BusToolTip } from './MapLayers/BusToolTip'
+import { BusToolTip } from './map-related/MapLayers/BusToolTip'
```

Comment:
why this import needed here?

Reply:
it's not. it should be removed. it's not related to this PR. it should be fixed

Reply:
you're the one to blame actually :sweat_smile: 
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/1d17caff-b947-4fdf-9aa7-89c97de7e5b6)

Reply:
Shame on me!

Reply:
no shame in [blame](https://git-scm.com/docs/git-blame)! :wink: